src/transformers/models/vit_mae/modeling_tf_vit_mae.py

```diff
@@ -969,50 +969,85 @@ def get_input_embeddings(self):
     def _prune_heads(self, heads_to_prune):
         raise NotImplementedError
 
-    def patchify(self, imgs):
+    def patchify(self, pixel_values):
         """
-        imgs: (batch_size, height, width, 3) x: (batch_size, num_patches, patch_size**2 *3)
+        Args:
+            pixel_values (`tf.Tensor` of shape `(batch_size, height, width, num_channels)`):
+                Pixel values.
```

Comment:
It seems that the MAE implementation in Tensorflow uses the channels last format.

cc @sayakpaul

Reply:
The model takes channels first input. But inside the embeddings layer, it transposes the inputs: 

https://github.com/NielsRogge/transformers/blob/68fb025fc26ab3e135affc55b590012cfa00dfce/src/transformers/models/vit_mae/modeling_tf_vit_mae.py#L343

This is is then propagated in the rest of the network hence `channels_last` Anything concerning?

Reply:
No, just wanted to confirm.